sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -131,17 +122,11 @@ export default abstract class HttpBaseClient {
    * @param payload data to be sent.
    * @param suppressErrorHandler optionally skip the global error handling.
    */
-  protected async patchCall<T>(
-    url: string,
-    payload: T,
-    suppressErrorHandler = false,
-  ): Promise<void> {
+  protected async patchCall<T>(url: string, payload: T): Promise<void> {
```

Comment:
Post has `config?: AxiosRequestConfig` and why patch doesn't have it?

Reply:
there is no scenario, as of now, I think if we any scenarios in future, we can add it in future

Reply:
![image](https://user-images.githubusercontent.com/77353155/212404994-821fa5ac-6da0-425f-84fe-df168ce5b904.png)